build/source/driver/multi_driver.f90

```diff
@@ -247,77 +357,123 @@ real(dp),parameter        :: safety = 0.01_dp               ! safety factor to e
  case default; call handle_err(30,'unable to identify vegetation category')
 end select
 
+! allocate space for default model parameters
+if(allocated(dparStruct%hru))then
+ call handle_err(20,'dparStruct is unexpectedly allocated')
+else
+ ! allocate the spatial dimension
+ allocate(dparStruct%hru(nHRU),stat=err)
+ if(err/=0) call handle_err(20,'problem allocating dparStruct')
```

Comment:
Above this is handled as a one liner. Should we handle all of them like that, e.g.

`allocate(computeVegFlux(nHRU),stat=err); call handle_err(err,'problem allocating space for computeVegFlux')`